src/main/scala/uk/gov/hmrc/ct/accounts/frs102/boxes/AC13.scala

```diff
@@ -28,9 +28,15 @@ case class AC13(value: Option[Int]) extends CtBoxIdentifier(name = "Turnover (pr
   with Validators {
 
   override def validate(boxRetriever: FullAccountsBoxRetriever): Set[CtValidation] = {
-    collectErrors(
+    val errors = collectErrors(
       validateInputAllowed("AC13", boxRetriever.ac205()),
-      validateMoney(value)
+      validateZeroOrPositiveInteger(this)
     )
+
+    if(errors.isEmpty) {
+      validateMoney(value, 0)
```

Comment:
I guess you could just have added the min = 0 param to the validateMoney above and not bothered with the validateZeroOrPositiveInteger as this would catch the negative case, but not a big deal. What do you think?